client/fingerprint/env_aws.go

```diff
@@ -494,6 +495,24 @@ func (f *EnvAWSFingerprint) Fingerprint(request *FingerprintRequest, response *F
 		}
 	}
 
+	// copy over IPv6 network specific information
+	if val, ok := response.Attributes["unique.platform.aws.mac"]; ok && val != "" {
+		k := "network/interfaces/macs/" + val + "/ipv6s"
+		addrsStr, err := ec2meta.GetMetadata(k)
+		if err != nil {
+			// don't log when the instance doesn't have an IPv6 address
+			awsErr, ok := err.(awserr.RequestFailure)
+			if !ok || awsErr.StatusCode() != 404 {
+				f.logger.Warn("could not read attribute value", "attribute", k, "error", err)
```

Comment:
In the other attributes, these get logged at `Debug`; we should be consistent here. It's probably a good idea to handle non-`awserr.RequestFailure` errors the same as we do above as well.

Reply:
I will change to `Debug` level.

For the non-`RequestFailure` case, are you suggesting essentially copying lines 456-467 to here, including the `return nil`?

Reply:
> For the non-RequestFailure case, are you suggesting essentially copying lines 456-467 to here, including the return nil?

Yes, thanks!

Reply:
Thanks for the feedback.  I have restructured it to handle errors the same as the `keys` loop.